file.go

```diff
@@ -39,8 +38,7 @@ type File struct {
 	delta   []*Undo // [private]
 	epsilon []*Undo // [private]
 	elog    Elog
-	name    string
-	info    os.FileInfo
+	*filedetails
```

Comment:
Composition over embedding please.

Reply:
I'm a little confused here because I thought that embedding is composition, are you saying to switch *filedetails to be details *filedetails?